libs/SalesforceAnalytics/src/com/salesforce/androidsdk/analytics/security/Encryptor.java

```diff
@@ -169,11 +211,11 @@ public static String decrypt(byte[] data, String key, byte[] iv) {
     }
 
     /**
-     * Encrypts data with key using AES-128.
+     * Encrypts data with key using AES/GCM/NoPadding.
      *
      * @param data Data.
      * @param key Base64 encoded 128 bit key or null (to leave data unchanged).
```

Comment:
Should say Base64 encoded 256 bit key

Reply:
Yes - I need to update all the comments.